apps/web/app/components/blocks/ImageGallery/ImageGalleryForm.vue

```diff
@@ -0,0 +1,110 @@
+      </div>
+
+      <div v-show="uiItemImageBlock.thumbnails.showThumbnails" class="space-y-3">
+        <UiFormLabel>{{ getEditorTranslation('thumbnail-type') }}</UiFormLabel>
+
+        <div class="grid grid-cols-2 gap-3">
+          <button
+            v-for="type in THUMBNAIL_TYPES"
+            :key="type"
+            type="button"
+            class="relative w-full rounded-lg border p-2 transition"
+            :class="
+              uiItemImageBlock.thumbnails.thumbnailType === type
+                ? 'border-neutral-900 ring-2 ring-neutral-900'
+                : 'border-neutral-300 hover:border-neutral-400'
+            "
+            :aria-pressed="uiItemImageBlock.thumbnails.thumbnailType === type"
+            :aria-label="typeLabels[type]"
+            data-testid="thumbnail-type-card"
+            @click="uiItemImageBlock.thumbnails.thumbnailType = type"
+          >
+            <NuxtImg :src="thumbnailsCdn[type]" :alt="typeLabels[type]" class="w-full h-auto rounded" />
+            <span
+              v-if="uiItemImageBlock.thumbnails.thumbnailType === type"
+              class="absolute -top-2 -right-2 inline-flex items-center justify-center w-6 h-6 rounded-full bg-neutral-900 text-white text-xs"
+              ><SfIconCheck size="xs"
+            /></span>
+          </button>
+        </div>
+      </div>
+
+      <div class="flex items-center justify-between">
+        <UiFormLabel>{{ getEditorTranslation('enable-zoom-on-hover') }}</UiFormLabel>
+        <SfSwitch v-model="uiItemImageBlock.thumbnails.enableHoverZoom" data-testid="enable-zoom-on-hover" />
+      </div>
+    </div>
+  </UiAccordionItem>
+</template>
+
+<script setup lang="ts">
+import { SfSwitch, SfIconCheck } from '@storefront-ui/vue';
+import { THUMBNAIL_TYPES } from '~/components/blocks/ImageGallery/types';
+import type { ThumbnailType, ImageGalleryContent } from '~/components/blocks/ImageGallery/types';
+
+type ItemImageFormProps = {
+  uuid?: string;
+};
+
+const props = defineProps<ItemImageFormProps>();
+
+const { data } = useCategoryTemplate();
+const { blockUuid } = useSiteConfiguration();
+const { findOrDeleteBlockByUuid } = useBlockManager();
+
+const uiItemImageBlock = computed(
+  () => findOrDeleteBlockByUuid(data.value, props.uuid || blockUuid.value)?.content as ImageGalleryContent,
+);
+
+const typeLabels: Record<ThumbnailType, string> = {
+  'left-vertical': getEditorTranslation('thumb-left-vertical'),
+  'right-vertical': getEditorTranslation('thumb-right-vertical'),
+  bottom: getEditorTranslation('thumb-bottom'),
+};
+
+const thumbsOpen = ref(true);
+
+const thumbnailsCdn = {
+  'left-vertical': 'https://cdn02.plentymarkets.com/v5vzmmmcb10k/frontend/PWA/Blocks/preview-thumbs-left.png',
+  'right-vertical': 'https://cdn02.plentymarkets.com/v5vzmmmcb10k/frontend/PWA/Blocks/preview-thumbs-right.png',
+  bottom: 'https://cdn02.plentymarkets.com/v5vzmmmcb10k/frontend/PWA/Blocks/preview-thumbs-bottom.png',
+};
```

Comment:
```suggestion
const thumbnails: Thumbnails = [
  {
    type: 'left-vertical',
    cdn: 'https://cdn02.plentymarkets.com/v5vzmmmcb10k/frontend/PWA/Blocks/preview-thumbs-left.png',
    label: getEditorTranslation('thumb-left-vertical'),
  },
  {
    type: 'right-vertical',
    cdn: 'https://cdn02.plentymarkets.com/v5vzmmmcb10k/frontend/PWA/Blocks/preview-thumbs-right.png',
    label: getEditorTranslation('thumb-right-vertical'),
  },
  {
    type: 'bottom-horizontal',
    cdn: 'https://cdn02.plentymarkets.com/v5vzmmmcb10k/frontend/PWA/Blocks/preview-thumbs-bottom.png',
    label: getEditorTranslation('thumb-bottom-horizontal'),
  },
];
```
In my opinion this is easier to read than having the information distributed across three separate definitions.